# Ejercicio1 - Utilizar operadores aritméticos
### Operadores aritméticos en Python

Exploremos cómo podemos crear un programa que pueda calcular la distancia entre dos planetas. Comenzaremos usando dos distancias de planetas: Tierra (149.597.870 km) y Júpiter (778.547.200 km).

In [3]:
# Crear variables para almacenar las dos distancias
# ¡Asegúrate de quitar las comas!

planet_E = 149597870
planet_J = 778547200

### Realizar la operación

Con los valores obtenidos, es el momento de añadir el código para realizar la operación. Restarás el primer planeta del segundo para determinar la distancia en kilómetros. A continuación, puedes convertir la distancia del kilómetro en millas multiplicándola por 0.621.

In [4]:
# Calcular la distancia entre planetas

dist_km = planet_J - planet_E
print(dist_km)

dist_mi = dist_km * 0.621
print(dist_mi)

628949330
390577533.93


### Prueba tu proyecto

Con el código creado, ejecuta el notebook para obtener el resultado. Deberías recibir lo siguiente:

628949330

390577534

# Ejercicio 2: convierte cadenas en números y usa valores absolutos

Para crear nuestra aplicación, queremos leer la distancia del sol para dos planetas, y luego mostrar la distancia entre los planetas. Haremos esto usando input para leer los valores, int para convertir a entero y luego abs para convertir el resultado en su valor absoluto.
Lee los valores

Usando input, agrega el código para leer la distancia del sol para cada planeta, considerando 2 planetas.


In [8]:
# Almacenar las entradas del usuario
# Pista: variable = input("¿Cuál es tu nombre?")

planet_E = input("Proporcione la distancia del sol para el planeta Tierra")
planet_J = input("Proporcione la distancia del sol para el planeta Jupiter")

### Convertir a número

Debido a que input devuelve valores de cadena, necesitamos convertirlos en números. Para nuestro ejemplo, usaremos int

In [9]:
# Convierte las cadenas de ambos planetas a números enteros

planet_E = int(planet_E)
planet_J = int(planet_J)

### Realizar el cálculo y convertir a valor absoluto

Con los valores almacenados como números, ahora puedes agregar el código para realizar el cálculo, restando el primer planeta del segundo. Debido a que el segundo planeta podría ser un número mayor, usarás abs para convertirlo a un valor absoluto. También agregarás el código para mostrar el resultado en millas multiplicando la distancia del kilómetro por 0.621

In [10]:
# Realizar el cálculo y determinar el valor absoluto

dist_km = planet_J - planet_E
print(dist_km)

# Convertir de KM a Millas

dist_mi = dist_km * 0.621
print(abs(dist_mi))

629000000
390609000.0


### Prueba tu aplicación
Para probar el proyecto, ejecuta tu notebook. En la parte superior de vscode surgirá un cuadro de diálogo para que proporciones las distancias. Puede utilizar los datos de la tabla siguiente:

| Planeta  | Distancia al sol |
| ------- | ----------------- |
| Mercurio | 57900000          |
| Venus   | 108200000         |
| Tierra   | 149600000         |
| Marte    | 227900000         |
| Júpiter | 778600000         |
| Saturno  | 1433500000        |
| Urano  | 2872500000        |
| Neptuno | 4495100000        |